package.json

```
@@ -2,13 +2,17 @@
   "name": "quickstart",
   "version": "0.0.0",
   "description": "oso Node.js quickstart",
-  "main": "server.js",
+  "main": "app.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/osohq/oso-nodejs/quickstart"
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "oso": ">=0.6.0"
+    "express": "~4.17.1",
+    "oso": ">=0.12.0"
+  },
+  "devDependencies": {
+    "supertest": "^6.1.3"
   }
```

```suggestion
  },
  "scripts":  {
      "start": "app.js"
  }
```

I needed this for it to run